SwiftWisdom/Core/StandardLibrary/String/String+EmailValidation.swift

```diff
@@ -63,15 +63,15 @@ extension String {
 
 
     /**
-     Validate if a password at least conforms to an array of requirements. For character types, 
-     requirments can define if they should be excluded or appear at least once. Minimum and maximum 
-     length are also requirment types. Does not define any expectation beyond what is in the array of 
+     Validate if a password at least conforms to an array of requirements. For character types,
+     requirments can define if they should be excluded or appear at least once. Minimum and maximum
+     length are also requirment types. Does not define any expectation beyond what is in the array of
      requirements (i.e. this function will always return true if the password requirement array is empty)
-     
+
```

Comment:
would be very nice if we added the parameter documentation, trival as it may be.